### PR TITLE
Create ui-c8y-1019-19-7-adjust-ui-info-gauge-widget-labels-6018-graft-release-cd-6049.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-57703](https://cumulocity.atlassian.net/browse/MTM-57703)
Original PR: [6049](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6049)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1019-19-7-adjust-ui-info-gauge-widget-labels-6018-graft-release-cd-6049.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: adjust UI info gauge widget labels (#6018) [GRAFT][release/cd] (#6049)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1019-19-7-adjust-ui-info-gauge-widget-labels-6018-graft-release-cd-6049.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Improved labels for info gauge widgets in the UI
```

File: content/change-logs/application-enablement/ui-c8y-1019-19-7-adjust-ui-info-gauge-widget-labels-6018-graft-release-cd-6049.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57703
+version: 1019.19.7
+---
+adjust UI info gauge widget labels (#6018) [GRAFT][release/cd] (#6049)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1019-19-7-adjust-ui-info-gauge-widget-labels-6018-graft-release-cd-6049.md:17]: The Claude AI suggested to use the following body:
```suggestion
The info gauge widgets in the UI previously had labels that were not fully descriptive or intuitive for users. To address this, the labels for these widgets have been updated to provide clearer information. This change improves the usability and readability of the info gauge widgets, making it easier for users to understand the data being presented without requiring additional explanation or context.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/application-enablement/ui-c8y-1019-19-7-adjust-ui-info-gauge-widget-labels-6018-graft-release-cd-6049.md (hunk 2 above)
Comment (eniosultan) [reply content/change-logs/application-enablement/ui-c8y-1019-19-7-adjust-ui-info-gauge-widget-labels-6018-graft-release-cd-6049.md:17]: @BeateRixen yeah adjust doesn't seem correct. maybe: "Labels in the Info gauge widget have been improved. Previously, labels with lengthy names encountered display issues. Now, adjustments have been made to ensure clearer and more accurate label representation."

File: content/change-logs/application-enablement/ui-c8y-1019-19-7-adjust-ui-info-gauge-widget-labels-6018-graft-release-cd-6049.md (hunk 1 above)
Comment (eniosultan) [inline content/change-logs/application-enablement/ui-c8y-1019-19-7-adjust-ui-info-gauge-widget-labels-6018-graft-release-cd-6049.md:3]:
```suggestion
title: Labels in the "Info gauge" have been improved.
```

File: content/change-logs/application-enablement/ui-c8y-1019-19-7-adjust-ui-info-gauge-widget-labels-6018-graft-release-cd-6049.md (hunk 2 above)
Comment (eniosultan) [inline content/change-logs/application-enablement/ui-c8y-1019-19-7-adjust-ui-info-gauge-widget-labels-6018-graft-release-cd-6049.md:17]:
```suggestion
Labels in the Info gauge widget have been improved. Previously, labels with lengthy names encountered display issues. Now, adjustments have been made to ensure clearer and more accurate label representation.
```

**Review by eniosultan — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1019-19-7-adjust-ui-info-gauge-widget-labels-6018-graft-release-cd-6049.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57703
+version: 1019.19.7
+---
+Labels in the Info gauge widget have been improved. Previously, labels with lengthy names encountered display issues. Now, adjustments have been made to ensure clearer and more accurate label representation.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1019-19-7-adjust-ui-info-gauge-widget-labels-6018-graft-release-cd-6049.md:17]:
```suggestion
Labels in the "Info gauge" widget have been improved. Previously, labels with lengthy names encountered display issues. Now, adjustments have been made to ensure clearer and more accurate label representation.
```

File: content/change-logs/application-enablement/ui-c8y-1019-19-7-adjust-ui-info-gauge-widget-labels-6018-graft-release-cd-6049.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Labels in the "Info gauge" have been improved.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1019-19-7-adjust-ui-info-gauge-widget-labels-6018-graft-release-cd-6049.md:3]:
```suggestion
title: Label display in the "Info gauge" widget has been improved
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**